packages/expo-notifications/README.md

```diff
@@ -0,0 +1,464 @@
+
+According to the [Android docs](https://developer.android.com/training/notify-user/channels),
+
+> You should create a channel for each distinct type of notification you need to send. You can also create notification channels to reflect choices made by users of your app. For example, you can set up separate notification channels for each conversation group created by a user in a messaging app.
+
+On devices with Android 7.1 and below, Expo will "polyfill" channels for you by saving your channel's settings and automatically applying them to any notifications you designate with the `channelId`.
+
+#### `Notifications.createChannelAsync(id: string, channel: Channel): Promise<void>`
+Creates channel.
+#### `Notifications.deleteChannelAsync(id: string): Promise<void>`
+Deletes channel.
+#### `Notifications.createChannelGroupAsync(groupId: string, groupName: string): Promise<void>`
+Creates channel group.
+#### `Notifications.deleteChannelGroupAsync(groupId: string): Promise<void>`
+Deletes channel group.
+
+### Scheduling
+
+#### `Notifications.scheduleNotificationWithCalendarAsync`
+
+##### Signature
+```ts
+async function scheduleNotificationWithCalendarAsync(
+  notification: ForegroundNotification,
+  options: {
+    year?: number;
+    month?: number;
+    hour?: number;
+    day?: number;
+    minute?: number;
+    second?: number;
+    weekDay?: number;
+    repeat?: boolean;
+  } = {}
+): Promise<string>
+```
+##### Description
+Schedules notification in cronlike style. 
+Returns notification id which later can be used to cancel or dismiss notification.
+
+#### `Notifications.scheduleNotificationWithTimerAsync()`
+##### Signature
+```ts
+async function scheduleNotificationWithTimerAsync(
+  notification: Notification,
+  options: {
+    interval: number;
+    repeat?: boolean;
+  }
+): Promise<string>
+```
+##### Description
+Presents notification after the specified number of seconds elapse.
+Returns notification id which later can be used to cancel or dismiss notification.
+
+#### `Notifications.cancelScheduledNotificationAsync(notificationId: string): Promise<void>`
+Cancels notification.
+#### `Notifications.cancelAllScheduledNotificationsAsync(): Promise<void>`
+Cancels all notifications.
+
+### Push Notifications
+
+#### `Notifications.setOnTokenChangeListenerAsync(listener: OnTokenChangeListener): Promise<void>`
+Sets token lister which will be triggered if push token is created or changed.
+If you choose to use Expo as a middleman then listener will receive ExpoPushToken and firebase token otherwise.
+Setting a token lister is also a registration for push notifications.
+
+### Other Methods
+
+#### `Notifications.dismissNotificationAsync(notificationId: string): Promise<void>`
+If sticky notification is diplayed you can dismissed it with this method.
+#### `Notifications.dismissAllNotificationsAsync(): Promise<void>`
+Dismiss All notifications.
+
+## Types
```

Comment:
Do we need Types section? Aren't they self explanatory if user uses any modern IDE? I guess that would make sense only if there is anything more than the code itself.

Reply:
Yeah, we need `types` section as not every IDE has this feature and sometimes (really often TBH) you check something online 😉